modules/ROOT/pages/architecture-visualization.adoc

```diff
@@ -0,0 +1,44 @@
+= Architecture Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== About the Architecture Visualization
+
+The Architecture Visualization shows the topology of your application network. In this visualization, you can change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.
```

Comment:
```suggestion
Architecture Visualization shows the topology of your application network. In this visualization, you can change display names of nodes, add tags, and assign nodes to layers. Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.
```